testing/adios2/bindings/python/CMakeLists.txt

```diff
@@ -4,9 +4,12 @@
 #------------------------------------------------------------------------------#
 
 if(NOT ADIOS2_HAVE_MPI)
-    python_add_test(PythonBPWrite TestBPWriteTypes_nompi.py)
+    python_add_test(TARGET PythonBPWrite SCRIPT TestBPWriteTypes_nompi.py)
```

Comment:
The `TARGET` option should be `NAME` since what your passing is the test name, not a CMake target.  The gtest functions actually used the CMake target information, but in this case, it's just a test name.